optimizely/config_manager.py

```diff
@@ -368,3 +368,76 @@ def start(self):
         """ Start the config manager and the thread to periodically fetch datafile. """
         if not self.is_running:
             self._polling_thread.start()
+
+
+class AuthDatafilePollingConfigManager(PollingConfigManager):
+    """ Config manager that polls for authenticated datafile based on access token. """
+
+    def __init__(
+        self,
+        access_token,
+        sdk_key=None,
+        datafile=None,
+        update_interval=None,
+        blocking_timeout=None,
+        url=None,
+        url_template=None,
+        logger=None,
+        error_handler=None,
+        notification_center=None,
+        skip_json_validation=False
```

Comment:
Done. I've made the change, but because there is no logic in `PollingConfigManager` to handle the `url_template` on the condition where an access token is passed, I've added a helper method in `AuthDatafilePollingConfigManager` to put the appropriate `url_template` in `kwargs` (depending on the args that are passed in to `AuthDatafilePollingConfigManager` constructor) before passing it into the parent class's constructor.